module-4/backend/week_two_cfu

```diff
@@ -0,0 +1,44 @@
+## Week Two - Module 4 Recap
+
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+
+Note: When you're done, submit a PR. 
+
+1. What is Webpack and why is it useful?
+* Webpack is middleware that manages the asset pipeline. It does such things as convert scss to css, converts ES6/ES7 to ES5 for browser compatibility, and minifies whitespace. Its essential function is to package up an application in a way that can be interpreted by the browser. 
+
+2. When do you want to use event delegation?
+* You want to use event delegation to reduce the amount of event listeners on the page. For instance, placing an event delegator on a table, when configured correctly, will allow survalence of events that occur within the table. This is more convienent than placing a lister on every row of the table, especially if the table is very large.
```

Comment:
absolutely. also - it's useful when we need events to be bound to elements that are dynamically loaded (via AJAX). we need to assign the event listener to a parent (thus using event delegation) so that the event will be bound!